android/app/src/main/java/im/status/ethereum/MainApplication.java

```diff
@@ -43,6 +46,10 @@ public boolean getUseDeveloperSupport() {
         protected List<ReactPackage> getPackages() {
             List<ReactPackage> packages = new ArrayList<ReactPackage>(Arrays.asList(
                     new MainReactPackage(),
+                    new UdpSocketsModule(),
```

Comment:
Is it related to firebase?

Reply:
This and TCP module imports were autogenerated - I believe as a leftover from when I targeted API 25. Removed and seems to build fine.